src/redux/Store.jsx

```diff
@@ -0,0 +1,17 @@
+/** ****************************************
```

Comment:
[OPTIONAL] I recommend to use lowercase for your redux files, it will help you to separate them from the React components easily 👍